docs/admin/limitrange/limits.yaml

```diff
@@ -4,20 +4,29 @@ metadata:
   name: mylimits
 spec:
   limits:
-  - max:
+  - default:
```

Comment:
remove this.  i will open a PR to ensure that default and defaultRequest on a a Pod LimitType results in validation error.